docs/rfc/0001-cli.md

```diff
@@ -0,0 +1,320 @@
+|     |
+|     +--> -o, [--out-file] FILENAME (Write SCC data into a file rather than into the DB)
+|     +--> -i, [--in-file] FILENAME (Load SCC data from a file rather than from SCC)
+|
++--+ products
+|  |
+|  +--+ list (List released products)
+|  |  |
+|  |  +--> -r RELEASE_STAGES (Show products with only specified stages: alpha, beta, released)
+|  |  +--> -c, [--custom-only], [--no-custom-only] (Show only custom products)
+|  |
+|  +--> enable PRODUCT (Enable a product)
+|  |
+|  +--> disable PRODUCT (Disable a product)
+|  |
+|  +--> create PRODUCT (Create a custom product)
+|  |
+|  +--> edit PRODUCT (Edit an existing custom product)
+|  |
+|  +--> delete PRODUCT (Delete a custom product)
+|  |
+|  +--> addrepo PRODUCT REPO_ID (Add a custom repository to the product)
+|  |
+|  +--> removerepo PRODUCT REPO_ID (Remove a custom repository from the product)
+|
++--+ repos
+|  |
+|  +--+ list [PRODUCT_STRING] (List repositories which are enabled for mirroring, optionally for a specified product)
+|  |  |
+|  |  +--> -a, [--all], [--no-all] (Show all repositories, default off)
+|  |  +--> -c, [--custom-only], [--no-custom-only] (Show only custom repositories)
+|  |
+|  +--+ enable TARGET (Enable repositories, TARGET is either product string (identifier/version/arch) or repo ID)
+|  |
+|  +--> disable TARGET (Disable repositories, TARGET is either product string (identifier/version/arch) or repo ID)
+|  |
+|  +--> create NAME URL (Create a custom repository)
+|  |
+|  +--> edit NAME URL (Edit an existing custom repository)
+|  |
+|  +--> delete TARGET (Delete a custom repository by name, ID or URL)
+|
++--> mirror (Mirror all repositories that are enabled)
+|
++--> help COMMAND (Show help for specified command)
+|
++--> version (Show RMT version)
+|
++--+ Global options:
+   |
+   +--> -h, [--help] (Another way to invoke `help` command)
+   +--> -v, [--version] (Another way to invoke `version` command)
+   +--> -d, [--debug] (Display debug output)
+
+```
+
+## Syncing with SCC
+
+```
+# rmt scc help
+Commands:
+  rmt scc sync     # Synchronize database with SCC
+
+  -o, [--out-file] FILENAME (Write SCC data into a file rather than into the DB)
+  -i, [--in-file] FILENAME (Load results from a file rather than from SCC)
+
+Run 'rmt scc help COMMAND' for more information on a command.
+```
+
+##### Respective SMT commands
+
+```
+smt-scc-sync
+```
+
+```
+smt-scc-sync : Get data from SCC and update the local database.
+
+Usage:
+smt-scc-sync [options]
+
+Options:
+--fromdir                 read SCC informations from directory instead of downloading it from SCC
+                          and update the database.
+--todir                   write SCC informations to directory without updating the database.
+--createdbreplacementfile create a database replacement file for using smt-mirror without database.
+--logfile -L file         Path to logfile
+--debug                   Enable debug mode
+--verboselevel -v level   set the verbose level
```

Comment:
usually just  --verbose

Reply:
That's SMT CLI (for reference), not RMT.